src/html/mod.rs

```diff
@@ -24,6 +24,10 @@ pub trait Component: Sized + 'static {
     type Properties: Properties;
     /// Initialization routine which could use a context.
     fn create(props: Self::Properties, link: ComponentLink<Self>) -> Self;
+    /// Called after the component has been attached to the VDOM and it is safe to receive messages
+    /// from agents. Any changes made to the state that `view` relies on will not be apparent
+    /// immediately as this method does not trigger a rerender.
+    fn on_mount(&mut self) {}
```

Comment:
What are your thoughts on naming this `mounted` instead? I think `on_mount` is clearest but `mounted` fits better with the existing trait methods

Reply:
I agree with that assessment.